src/types.ts

```diff
@@ -50,6 +53,7 @@ export interface Marketplace {
   description: string
   logoUrl: string
   bannerUrl: string
+  auctionHouses?: AuctionHouse[]
   auctionHouse: AuctionHouse
```

Comment:
Lets drop auctionHouse from this version of the type since we are migrating everything to auctionHouses.